source/_sass/page.scss

```diff
@@ -1,333 +1,330 @@
+    @media #{$x-large} {
+      @include grid(12,4.5);
+      @include prefix(12,2);
+    }
+  }
 }
 #index {
-	@include container;
-	@include grid(12,10);
-	@include prefix(12,1);
-	@include suffix(12,1);
-	margin-bottom: 2em;
-	@media #{$small} {
-		@include grid(12,6);
-		@include prefix(12,0.5);
-		@include suffix(12,0.5);
-	}
-	@media #{$x-large} {
-		@include grid(12,4.5);
-		@include prefix(12,2);
-	}
-	h3 {
-		margin: 0;
-		padding-bottom: .5em;
-		@include font-rem(28);
-		border-bottom: 1px solid lighten($black,70);
-	}
-	article {
-		h2 {
-			margin-bottom: 4px;
-			@include font-rem(20);
-			&.link-post {
-				margin-bottom: 0px + $doc-line-height;
-				margin-bottom: 0rem + ($doc-line-height / $doc-font-size);
-			}
-		}
-		p {
-			@include font-rem(14);
-		}
-		p+p {
-			text-indent: 0;
-		}
-	}
+  @include container;
+  @include grid(12,10);
+  @include prefix(12,1);
+  @include suffix(12,1);
+  margin-bottom: 2em;
+  @media #{$small} {
+    @include grid(12,6);
+    @include prefix(12,0.5);
+    @include suffix(12,0.5);
+  }
+  @media #{$x-large} {
+    @include grid(12,4.5);
+    @include prefix(12,2);
+  }
+  h3 {
+    margin: 0;
+    padding-bottom: .5em;
+    @include font-rem(28);
+    border-bottom: 1px solid lighten($black,70);
+  }
+  article {
+    h2 {
+      margin-bottom: 4px;
+      @include font-rem(20);
+      &.link-post {
+        margin-bottom: 0px + $doc-line-height;
+        margin-bottom: 0rem + ($doc-line-height / $doc-font-size);
+      }
+    }
+    p {
+      @include font-rem(14);
+    }
+    p+p {
+      text-indent: 0;
+    }
+  }
 }
 .image-wrap {
-	position: relative;
-	margin-bottom: 2em;
-	@include clearfix;
-	&:after {
-		content: " ";
-		display: block;
-		position: absolute;
-		bottom: 0;
-		left: 8%;
-		width: 0;
-		height: 0;
-		border: 10px solid transparent;
-		border-bottom-color: $bodycolor;
-		@media #{$small} {
-			left: 25%;
-		}
-		@media #{$large} {
-			border-width: 20px;
-		}
-		@media #{$x-large} {
-			left: 33.333333333%;
-		}
-	}
-	img {
-		width: 100%;
-		height: auto;
-		-ms-interpolation-mode: bicubic;
-	}
-	.headline-wrap {
-		position: absolute;
-		bottom: 25px;
-		@include prefix(12,1);
-		@media #{$small} {
-			left: 25%;
-			margin-left: 0;
-		}
-		@media #{$x-large} {
-			@include prefix(12,1);
-		}
-		h1, h2 {
-			color: $white;
-			@include text-shadow(0 1px 2px rgba(0,0,0,.25));
-		}
-	}
+  position: relative;
+  margin-bottom: 2em;
+  @include clearfix;
+  &:after {
+    content: " ";
+    display: block;
+    position: absolute;
+    bottom: 0;
+    left: 8%;
+    width: 0;
+    height: 0;
+    border: 10px solid transparent;
+    border-bottom-color: $bodycolor;
+    @media #{$small} {
+      left: 25%;
+    }
+    @media #{$large} {
+      border-width: 20px;
+    }
+    @media #{$x-large} {
+      left: 33.333333333%;
+    }
+  }
+  img {
+    width: 100%;
+    height: auto;
+    -ms-interpolation-mode: bicubic;
+  }
+  .headline-wrap {
+    position: absolute;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets